app/wp-content/themes/kstg/library/scss/components/_typography.scss

```diff
@@ -6,31 +6,55 @@ body {
   color: $dark-grey;
 }
 
-// Headers
 h1 {
-  margin: 0 10%;
-  padding-bottom: 20px;
-  font-size: 26px;
+  font-size: 29px;
   font-weight: bold;
   text-align: center;
-  line-height: 30px;
+  color: $purple-text;
 }
 
+//body headlines and navigation
 h2 {
-  font-size: 26px;
-  font-weight: bold;
-}
-
-h2.h2-center {
-  text-align: center;
+  font-size: 20px;
+  font-weight: 700;
 }
 
 h3 {
   font-size: 16px;
   font-weight: bold;
-  padding: 20px 0 3px 0;
+  padding: 20px 0 10px 0;
 }
 
 p {
   font-size: 14px;
+  color: $dark-grey;
+}
+
+// Headline 1  -  28 pt
+// Headeline 2  -  20 pt  (body headlines & navigation)
+// Body copy  -  18 pt
+// Dropdown menu copy - 16 pt
+// Footer copy  - 14 pt
+// Breadcrumb  -  12 pt
+//MOBILE
+// Headline 1 - 18 pt
+// Headline 2 - 18 pt (body headlines)
+// Body copy - 16 pt
+// Navigation menu & footer copy - 14 pt
+
+/************/
+/** MOBILE **/
+/************/
+@media only screen and (max-width: 460px) {
+  h1 {
```

Comment:
One thing you might want to consider to save you time in the long run is using 'ems' for font size. The nice think about them is that they cascade. Meaning, if you apply a certain size to the body of the page, all font sizes will fall in line (you don't have to go through and manually change all of your `h1`s, all of your `p`s, etc.).

Here's a good CSS-Tricks article about them: https://css-tricks.com/why-ems/